Add docs of AzureChaos (#287)
chaos-mesh/website: docs/simulate-azure-chaos.md

   ```bash
   kubectl apply -f azurechaos-disk-detach.yaml
   ```

### Field description

The following table shows the fields in the YAML configuration file.

| Parameter | Type | Description | Default value | Required | Example |
| --- | --- | --- | --- | --- | --- |
| action | string | Indicates the specific type of faults. Only vm-stop, vm-restart, and disk-detach are supported. | vm-stop | Yes | vm-stop |
| mode | string | Specifies the mode of the experiment. The mode options include `one` (selecting a random Pod), `all` (selecting all eligible Pods), `fixed` (selecting a specified number of eligible Pods), `fixed-percent` (selecting a specified percentage of Pods from the eligible Pods), and `random-max-percent` (selecting the maximum percentage of Pods from the eligible Pods). | None | Yes | `one` |
| value | string | Provides parameters for the `mode` configuration, depending on `mode`.For example, when `mode` is set to `fixed-percent`, `value` specifies the percentage of Pods. | None | No | 1 |
| secretName | string | Specifies the name of the Kubernetes Secret that stores the Azure authentication information. | None | No | cloud-key-secret |

> | subscriptionID | string | Specifies the VM instacnce's subscription ID | None | Yes | your-subscription-id |

Reviewer: ```suggestion
| subscriptionID | string | Specifies the VM instacnce's subscription ID. | N/A | Yes | `your-subscription-id` |
```

> | resourceGroupName | string | Specifies the Resource group of VM | None | Yes | your-resource-group-name |

Reviewer: ```suggestion
| resourceGroupName | string | Specifies the Resource group of VM. | N/A | Yes | `your-resource-group-name` |
```

> | lun | string | This is a required field when the `action` is `disk-detach`, specifies the LUN(Logic Unit Number) of data disk | None | No | `0` |

Reviewer: ```suggestion
| lun | string | This is a required field when the `action` is `disk-detach`, specifies the LUN (Logic Unit Number) of data disk. | N/A | No | `0` |
```

> | diskName | string | This is a required field when the `action` is `disk-detach`, specifies the name of data disk | None | No | `DATADISK_0` |

Reviewer: ```suggestion
| diskName | string | This is a required field when the `action` is `disk-detach`, specifies the name of data disk. | N/A | No | `DATADISK_0` |
```

> | duration | string | Specifies the duration of the experiment. | None | Yes | 30s |

Reviewer: ```suggestion
| duration | string | Specifies the duration of the experiment. | N/A | Yes | `30s` |
```
